testapi.pm

```diff
@@ -1529,8 +1461,8 @@ You can pass the same optional parameters as for C<type_string> function.
 
 =cut
 
-sub enter_cmd {
-    type_string shift, lf => 1, @_;
+sub enter_cmd ($string, @) {
+    type_string $string, lf => 1, @_;
```

Comment:
`@_`

Reply:
I guess `# no:style:signatures` addresses that (you only see that in the full diff now)